README.md

```diff
@@ -1,33 +1,105 @@
-# GitHub action to auto-assign issues to team members based on their availability and busyness
+# GitHub Action(s) to help automatically assign issues to the right persons
 
-This repository contains 2 actions which may be useful to someone assigning issues to dedicated engineering teams and distributing (automatically assigning) them based on the availability of individual engineers.
+This repository contains 2 actions which are useful to someone assigning issues to dedicated engineering teams and distributing (automatically assigning) them based on the availability of individual engineers:
 
 ## Regex labeler
 
-This github action allows to assign labels based on regular expressions which are ranked against the issue title and body it's run against. This allows to assign team specific labels by matching Issues against these regex's. 
+This action allows to assign labels based on regular expressions which are ranked against the issue title and body it's run against. This allows to assign team specific labels by matching issues against these regex's.
+
+### Inputs
+
+| Parameter  | Type   | Required | Default                     | Description                                |
+| ---------- | ------ | -------- | --------------------------- | ------------------------------------------ |
+| `gh-token` | String | false    | `${{ github.token }}`       | The github token to be used for API access |
+| `cfg-path` | String | true     | `.github/regex-labeler.yml` | Path for regex labeler config file         |
+
+### Configuration
+
+An exemplary configuration looks like this: 
+
+```yaml
+required_labels:
+labels:
+  regex-labeler:
+    matchers:
+      - regex: "regex"
```

Comment:
i think this is a confusing example because the regular expression is "regex", which doesn't make much sense, a regular expression doesn't always have the string "regex" in it.

how about another example that's more obvious to understand:

```suggestion
  numeric-labeler:
    matchers:
      - numeric: "[0-9]+"
```